### DISCUSSION TITLE
Melhor curso para desenvolvimento web?

### DISCUSSION THREAD

**Discussion by LSantiago54** [frontendbr/forum#2361] [Perguntas e Respostas]
Esse ano decidi começar a aprender programação. Comecei pelo curso do Programador BR, mas de um tempo pra ca estou sentido que o curso peca muito no quesito exercício.  É muita pouca pratica. Pra ter uma ideia,  conheci o gitHub recentemente , e junto com ele o frontendmentor. Em 2 projetos eu fixei mais conteudo do que em 3 meses de aula.
Por isso gostaria da opinião de vcs, que curso vcs indicam pra um iniciante em desenvolvimento web? Um que tenha bastante pratica com exercícios e etc.

**Comment by Diego-Girao** [comment]
Mano, tem a DIO com bootcamps, Rocketseat, Alura, Origamid tem um curso muito bom e barato o plano vitalicio, tem tambem o Guanabara no Youtube, DevMedia  bom preço e é Gameficado o conteúdo, HCode Lab / Cafe.

**Comment by rafaeldellaquila** [comment]
Cara, na Udemy tem muita coisa boa, gosto muito dos cursos do Willian Justen e do Luiz Otávio Miranda. Origamid também é uma plataforma ótima para isso, o @Diego-Girao falou da Alura mas acho bem caro para o que oferece, sinceramente...

**Comment by wilsonneto-swe** [comment]
Segue um artigo que escrevi com indicacoes de cursos: https://www.javascriptnews.com.br/post/melhores-cursos-de-js-react-node-e-typescript

**Comment by gustvoassuncao** [comment]
Cara eu fiz bootcamp da Labenu, mas para mim que queria resultados mais rápidos, eu fiz o integral de 6 meses, foi pesado pra caramba e pude evoluir muito, mas só recomendo se estiver preparado para abdicar de várias coisas, a carga é pesada pra caramba, eu estudava os três turnos e final de semana tb, mas valeu muito a pena pra mim! Agora se for cursos de aulas gravadas eu recomendo da Rockectseat ou o do b7Web! Qualquer coisa, só falar!
